Tools/WinMLRunner/src/BindingUtilities.cpp

```diff
@@ -646,6 +646,28 @@ namespace BindingUtilities
         }
     }
 
+    TensorString CreateTensorString(const std::vector<int64_t>& tensorShape)
+    {
+        auto length = 1;
+        for (auto&& dim : tensorShape)
+        {
+            if (dim > 0)
+            {
+                length *= dim;
+            }
+        }
+
+        std::vector<winrt::hstring> strings;
+
+        for (auto i = 0; i < length; ++i)
+        {
+            auto str = to_hstring(i);
```

Comment:
I'm not sure I understand what this logic is supposed to do. Why do we want to create a list of strings that looks like `{"0", "1", "2", ..., "100", "101", "102"}`?

Reply:
it's just a way to create valid strings. They don't have any meaning, but they are unique, valid strings to pass to the model. Basically garbage data like the other garbage data generating methods.